EIPS/eip-9999.md

```diff
@@ -0,0 +1,84 @@
+---
+eip: 
+title: Anti-correlation attestation penalties
+description: Adjust penalties for missed attestations based on in-slot correlation of missed attestation
+author: dapplion (@dapplion), Toni Wahrstätter (@nero_eth), Vitalik Buterin (@vbuterin)
```

Comment:
```suggestion
author: dapplion (@dapplion), Toni Wahrstätter (@nerolation), Vitalik Buterin (@vbuterin)
```
Toni's GitHub: https://github.com/Nerolation

Reply:
Thanks for fixing that. I'm planning to move to nero_eth in the near future - will change it as soon as I'm ready.